unittests/v1.2.0/test_report_generation.py

```diff
@@ -0,0 +1,87 @@
+from compliance_suite.report_runner import *
+
+auth_types = ["none", "basic", "bearer", "passport"]
+
+def test_constructor():
+    tvr = ValidateResponse()
+    assert tvr.actual_response == ""
+    assert tvr.expected_response == ""
+    assert tvr.response_schema_file == ""
+    assert tvr.case == ""
+    
+def test_report_runner():
+
+    port_numbers = [8089, 8090, 8091, 8092]
+
+    for authtype, port in zip(auth_types, port_numbers):
+        good_report = report_runner(server_base_url = "http://localhost:" + str(port) + "/ga4gh/drs/v1",
+                                    platform_name = "good mock server",
+                                    platform_description = authtype,
+                                    drs_version = "1.2.0",
+                                    config_file = "compliance_suite/config/config_samples/config_" + authtype + ".json")
+
+        good_json = json.loads(good_report.to_json())
+
+        # remove timestamps, otherwise assert will fail 100%
+        good_json["start_time"] = ""
+        good_json["end_time"] = ""
+        for phase in good_json["phases"]:
+            phase["start_time"] = ""
+            phase["end_time"] = ""
+            for test in phase["tests"]:
+                test["start_time"] = ""
+                test["end_time"] = ""
+                for case in test["cases"]:
+                    case["start_time"] = ""
+                    case["end_time"] = ""
+                    assert case["status"] == "PASS"
```

Comment:
After spending some time on this code, I recommend revising our approach. Instead of performing a blanket check for status = PASS, we should account for the specific scenario where status = SKIP. I suggest we either address the case where status = SKIP or remove this assertion altogether.